dumper/dataWritter.go

```diff
@@ -277,6 +304,48 @@ func filterRowData(value []rowDataStructure, table schemareader.Table) []rowData
 	}
 	return value
 }
+func buildQueryToGetExistingRecords(path []string, table schemareader.Table, schemaMetadata map[string]schemareader.Table) string {
+	mainUniqueColumns := strings.Join(table.UniqueIndexes[table.MainUniqueIndexName].Columns, ",")
+	joinsClause := getJoinsClause(path, table, schemaMetadata)
+	whereClause := fmt.Sprintf(`WHERE rhnchannel.id = (SELECT id FROM rhnchannel WHERE label = %s`, "replace-me)")
```

Comment:
Is this the intention in here? We will not be using the channel label

Reply:
It was intentional. I wanted to check if you guys had any objection to handling channels in sequence. So create all the insert statements for 1 channel and once it's done then handle next channel.

Reply:
@rjmateus I have added a new separate commit https://github.com/uyuni-project/inter-server-sync/pull/6/commits/b53b8bafc433e00f220190db9dad435bd59e46b6 so we move to the next channel processing only if the current one is done.